shopinvader_sale_packaging/services/abstract_sale.py

```diff
@@ -27,9 +27,7 @@ def _convert_one_line(self, line):
         if line.product_packaging:
             pkg_vals.update(
                 {
-                    "packaging": line.product_packaging.jsonify(
-                        ["id", "name"]
-                    )[0],
+                    "packaging": line.product_packaging.jsonify(["id", "name"])[0],
```

Comment:
**Tips:** you can use the `one=True` parameter
https://github.com/OCA/server-tools/blob/14.0/base_jsonify/models/models.py#L93

```suggestion
                    "packaging": line.product_packaging.jsonify(["id", "name"], one=True),
```